lint-staged.config.js

```diff
@@ -1,9 +1,7 @@
 const lintStagedConfig = {
 	// run formatting and linting on all supported file types
-	"*.{js,json,md,ts,yaml,yml}": "npm run format:write",
-	"*.{js,ts}": ["npm run lint:fix"],
-	// run testing on all supported file types within the src/ directory
-	"src/**/*.{js,ts}": ["npm run test -- --findRelatedTests"],
+	"*.{js,json,md,ts,yaml,yml}": "npm run format:check",
+	"*.{js,ts}": ["npm run lint"],
```

Comment:
Why are we no longer automating fixing lint issues as part of `lint-staged`?

Reply:
As discussed on a call, decided to revert it to fix lint errors and formatting as a part of pre-commit hook